xml/admin_ceph_troubleshooting.xml

```diff
@@ -515,4 +515,101 @@ keep_jobs: 1</screen>
    </procedure>
   </tip>
  </sect1>
+ <sect1 xml:id ="ceph-fix-too-many-pgs">
+   <title>Too Many PGS Per OSD</title>
+   <para>
+     The <option>TOO_MANY_PGS</option> flag is raised when the number of PGs in
+     use is above the configurable threshold of <option>mon_pg_warn_max_per_osd</option>
+     PGs per OSD. If this threshold is exceeded, the cluster does not allow new pools
+     to be created, pool <option>pg_num</option> to be increased, or pool
+     replication to be increased.
+   </para>
+   <para>
+     &productname; &prevproductnumber; and &productnumber; have two ways to
+     solve this issue.
+   </para>
+   <procedure>
+     <title>Solution 1</title>
+     <step>
+       <para>
+         Set the following in your <filename>ceph.conf</filename>
+       </para>
+<screen>
+[global]
+
+mon_max_pg_per_osd = 800  #  depends on you amount of PGs
+osd max pg per osd hard ratio = 10 #  default is 2, try to set at least 5. It will be
```

Comment:
"It will be" what?

Also, comma splice.

Reply:
LOL thanks.

Reply:
More general question:

I get why in some contexts one would use inline comments in config files, but if one is citing a config file or worse still a command in documentation, would it not be preferable (& remove a potential cause of errors or misunderstandings) to use a callout or something rather than include code commands in a PDF, webpage or worst of all paper?